socorro/external/es/super_search_fields.py

```diff
@@ -192,6 +192,26 @@ def get_connection(self):
         with self.context() as conn:
             return conn
 
+    def get_supersearch_status(self):
+        """Return list of indices, latest index, and mapping.
+
+        :returns: dict with keys "indices", "latest_index", "mapping"
+
+        """
+        conn = self.get_connection()
+        index_client = elasticsearch.client.IndicesClient(conn)
+        indices = self.context.get_indices()
+        latest_index = sorted(indices)[-1]
```

Comment:
It's possible if Socorro is creating indexes in the future that the latest index isn't the current index. I'm going to ignore that for now but keep it in the back of my head.